### ISSUE TITLE
mp4box fails to handle video in mp4 file

### ISSUE THREAD

**Issue by alexrayne** [gpac/gpac#967]
Hallow !
use gpac 0.7.0 on win8.1
i make mp4 video by avidemux with mpeg4 ASP xvid4 codec.
produced file i try to demux video trek, and mp4box fails with no messages, and stop by windows.
here is my video https://cloud.mail.ru/public/4Szu/TJS3H3Asn
command to reproduce:
"c:\util\video\gpac\mp4box.exe" -add "20171218(200949)-lr.mp4#1:fps=25.000" -itags tool="Yamb 2.1.0.0 [http://yamb.unite-video.com]" -new "20171218(200949)-lraac.mp4"

**Comment by aureliendavid** [comment]
your video link isn't working, can you post it elsewhere ? (e.g. here https://www.mediafire.com/filedrop/filedrop_hosted.php?drop=eec9e058a9486fe4e99c33021481d9e1826ca9dbc242a6cfaab0fe95da5e5d95
)

**Comment by alexrayne** [comment]
I`ve just now open it well with Chrome on Win8

**Comment by aureliendavid** [comment]
Hi @alexrayne 

I managed to download your first file from a different network. The second link (on gdrive) is a different file that works fine. 

I reproduced your issue with the first file, the issue came from an invalid m4v track (it's missing a decoderSpecificInfo box).

I fixed the crash so that the -add command should now work, however it will copy the invalid track as is, which might cause problems down the line. 

If that's not enough, another solution is to extract the raw track : 

`MP4Box -raw 1 "20171218(200949)-lr.mp4"`

and then reimport it to a new file : 

`MP4Box -add "20171218(200949)-lr_track1.cmp" -new "20171218(200949)-lr_track1.mp4"`

mp4box should create the missing data to produce a valid file.
